solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -84,44 +88,27 @@ for ip in $PUBLIC_IP;do rsync -avzrP -e 'sudo ssh -i cortx.pem -o StrictHostKeyC
 
 **Note:** Become the **root** user after logged in to the primary node by running `sudo su` command.
 ```
-AWS_PRIMARY_IP=$(cat ip_public.txt | jq '.[0]'| tr -d '",[]')
-ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@$AWS_PRIMARY_IP
-git clone https://github.com/Seagate/cortx-re && cd $PWD/cortx-re/solutions/community-deploy
-time bash -x ./build-cortx.sh
-```
-- Save and compress the cortx build images by running following command,
-```
-echo "Wait till the operation is completed..." && docker save $(docker images | sed '1d' | awk '{print $1 ":" $2 }') -o /tmp/cortximages.tar
-```
-- Execute the following command to copy the cortx build images from **primary node to all the worker nodes using private ip address**,
+export PRIMARY_IP=$(cat ip_public.txt | jq '.[0]'| tr -d '",[]')
+ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@$PRIMARY_IP "git clone https://github.com/Seagate/cortx-re && cd $PWD/cortx-re/solutions/community-deploy && time bash -x ./build-cortx.sh"
 ```
-AWS_WORKER_IP=$(cat /tmp/ip_private.txt | jq '.[1]','.[2]' | tr -d '",[]')
-for worker in $AWS_WORKER_IP;do cd /tmp && rsync -avzrP -e 'sudo ssh -i cortx.pem -o StrictHostKeyChecking=no' /tmp/*.tar /etc/docker/daemon.json centos@$worker:/tmp; done
-```
-- Load the cortx build images to the worker nodes by executing the following command,
-```
-for worker in $AWS_WORKER_IP;do ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@$worker 'sudo docker load -i /tmp/cortximages.tar'; done
-```
-- Clone cortx-re repository from required branch/tag. If you do not provide `-b <branch/tag>`, then it will use default main branch    
-  :warning: Tag based build is supported after including tag [2.0.0-879](https://github.com/Seagate/cortx-re/releases/tag/2.0.0-879)
+- Clone cortx-re repository from required branch/tag. If you do not provide `-b <branch/tag>`, then it will use default `main` branch    
+  :warning: Tag based build is supported after including tag [2.0.0-940](https://github.com/Seagate/cortx-re/releases/tag/2.0.0-940)
```

Comment:
We should instruct users to use the latest release from https://github.com/Seagate/cortx/releases instead of hardcoded `2.0.0-940`